### PR TITLE
Add some more topics and keywords to pyproject.toml

### PR DESCRIPTION
**Pull request by michelcrypt4d4mus**
Never hurts to classify the package more for SRO etc. happy to remove or modify anything here (or just close if you don't want this)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -8,12 +8,24 @@ version = "5.0.0"
...
+    "D-Bus",
+    "Desktop Bus",
+    "desktop-notifier",
+    "notifications",
+    "Notification Center",
+    "push notifications",
+    "Toast notifications",
+]
 classifiers = [
     "Development Status :: 2 - Pre-Alpha",
```
Comment (michelcrypt4d4mus) [inline pyproject.toml:21]: @samschott you might want to change this as `Pre-Alpha` could be pretty off putting to people searching around for libraries on pypi

File: pyproject.toml [hunk 2]
```diff
@@ -8,12 +8,24 @@ version = "5.0.0"
 authors = [{name = "Sam Schott", email = "sam.schott@outlook.com"}]
 license = {text = "MIT"}
 description = "Python library for cross-platform desktop notifications"
-keywords = ["desktop-notifier"]
+keywords = [
+    "D-Bus",
+    "Desktop Bus",
```
Comment (samschott) [inline pyproject.toml:13]: IMO, D-Bus and Desktop Bus are slightly misleading keywords which may suggest that this is a some kind of dbus library, whereas desktop notifier just uses it as a communication channel.

**Review by samschott — CHANGES_REQUESTED** [review]
Thanks! Small comment inline,

File: pyproject.toml (hunk 2 above)
Comment (michelcrypt4d4mus) [reply pyproject.toml:13]: happy to delete it; whatever you think. my thinking was just that someone who is familiar with linux enough to be using a notifications app would theoretically know that such a library would have to use D-Bus? but tbh i don't know anything about D-Bus (literally never heard of it) or really any kind of linux desktop environment.

File: pyproject.toml (hunk 1 above)
Comment (samschott) [reply pyproject.toml:19]: Good point!

Comment (samschott) [reply pyproject.toml:19]: Feel free to replace it with `Development Status :: 5 - Production/Stable` in this PR.

File: pyproject.toml (hunk 2 above)
Comment (samschott) [reply pyproject.toml]: Yeah, I think it's better to remove those two.

**Review by samschott — APPROVED** [review]

**Pull request merged**